packages/orbit-components/src/Accordion/AccordionSection/index.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from "react";
+
+import { useAccordion } from "../AccordionContext";
+import randomID from "../../utils/randomID";
+import useBoundingRect from "../../hooks/useBoundingRect";
+import Slide from "../../utils/Slide";
+import AccordionWrapper from "../components/AccordionWrapper";
+import SectionHeader from "./components/SectionHeader";
+import SectionFooter from "./components/SectionFooter";
+import SectionContent from "./components/SectionContent";
+
+import type { Props } from "./index";
+
+const AccordionSection = ({
+  children,
+  header,
+  footer,
+  actions,
+  dataTest,
+  expandable = true,
+}: Props) => {
+  const { expanded, onExpand } = useAccordion();
+
+  const slideID = React.useMemo(() => randomID("slideID"), []);
+  const labelID = React.useMemo(() => randomID("labelID"), []);
+
+  const [{ height }, ref] = useBoundingRect({ height: expanded ? null : 0 });
+
+  return (
+    <AccordionWrapper dataTest={dataTest}>
+      {header && (
+        <SectionHeader
+          actions={actions}
+          expanded={Boolean(expanded)}
```

Comment:
Is typecasting here necessary considering that you're enforcing the type with a static type, and that `expanded` is required?
```suggestion
          expanded={expanded}
```

Reply:
I suggested removing the typecasting because (the way I see it) the value of `expanded` is already boolean, do you disagree then?